### PR TITLE
Improve system start service

### PR DESCRIPTION
**Pull request by anton-bondarev**
* Enable different ttys
* Enable output commands during starting

### CODE REVIEW

File: src/init/system_start_service.c [hunk 1]
```diff
@@ -23,6 +24,16 @@ static const char *script_commands[] = {
...
+#if OPTION_GET(NUMBER,log_level) >= LOG_INFO
+#	define MD(x) do {\
+		x;\
+	} while (0);
+#else
+#	define MD(x) do{\
+	} while (0);
+#endif
+
+#define PRINTF printf
```
Comment (Deryugin) [inline src/init/system_start_service.c]: This macro is not used

File: src/init/system_start_service.c [hunk 2]
```diff
@@ -32,6 +43,7 @@ int system_start(void) {
 	setup_tty(OPTION_STRING_GET(tty_dev));
 
 	array_foreach(command, script_commands, ARRAY_SIZE(script_commands)) {
+		MD(printf(">%s\n", command));
```
Comment (Deryugin) [inline src/init/system_start_service.c]: May be `log_boot()` can be used here?

File: src/init/system_start_service.c (hunk 1 above)
Comment (anton-bondarev) [reply src/init/system_start_service.c]: Thanks, fixed

File: src/init/system_start_service.c (hunk 2 above)
Comment (anton-bondarev) [reply src/init/system_start_service.c]: No, I don't think so. it's a first user process, it should use printf in my mind

File: templates/x86/qemu/mods.config [hunk 3]
```diff
@@ -94,8 +94,8 @@ configuration conf {
 	@Runlevel(1) include embox.test.util.hashtable_test
 
 	@Runlevel(2) include embox.cmd.sh.tish(prompt="%u@%h:%w%$", rich_prompt_support=1, builtin_commands="exit logout cd export mount umount")
-	@Runlevel(2) include embox.init.start_script(shell_name="tish", tty_dev="ttyS0", shell_start=1, stop_on_error=true)
-	include embox.init.system_start_service
+	//@Runlevel(2) include embox.init.start_script(shell_name="tish", tty_dev="ttyS0", shell_start=1, stop_on_error=true)
```
Comment (Deryugin) [inline templates/x86/qemu/mods.config]: Is it necessary to leave this line?

**Pull request merged**